spec/factories/budget.rb

```diff
@@ -1,7 +1,7 @@
 FactoryBot.define do
   factory :budget do
-    budget_type { "original" }
```

Comment:
It feels like a shame to loose the clarity in the code of what 1 and 2 represent, do you think we can use a presenter for the xml that renders the values as 1 and 2 but we keep seeing 'original' and 'indicative'?

Reply:
I've done it with i18n as I'm not sure how to have it in a presenter and not mess up the view elsewhere. I'm not super hot on presenters but I think they need to have method names the same as the attribute name, so having something like `BudgetPresenter.iati_status` wouldn't work (I think??)